tools/figma-inspector/tests/property-parsing.test.ts

```diff
@@ -68,7 +70,19 @@ function processCornerRadii(json: any): any {
 
     return json;
 }
-
+function processVectorNode(json: any): any {
+    // You can expand this as needed for your test expectations
+    if (json.type === "VECTOR") {
+        return {
+            ...json,
+            vectorPaths: json.vectorPaths || [],
+            strokes: json.strokes || [],
+            strokeWeight: json.strokeWeight ?? 1,
+            exportAsync: async () => `<svg><path d="M10 10L90 90"/></svg>`,
```

Comment:
For this to work, I think you need to use a `return` statement. So `return "<svg....";`.